recipes/freeglut/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class freeglutConan(ConanFile):
+    name = "freeglut"
+    version = "3.2.1"
+    description = "Open-source alternative to the OpenGL Utility Toolkit (GLUT) library"
+    topics = ("conan", "freeglut", "opengl", "gl", "glut", "utility", "toolkit", "graphics")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dcnieho/FreeGLUT"
+    license = "X11"
```

Comment:
That's new :eyes:

Reply:
what exactly? X11 license?

Reply:
MIT Flavor, it contains one more paragraph:

> Except as contained in this notice, the name(s) of the above copyright holders shall not be used in advertising or otherwise to promote the sale, use or other dealings in this Software without prior written authorization.